pkg/mco/mcplist.go

```diff
@@ -0,0 +1,138 @@
+package mco
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func ListMCP(apiClient *clients.Settings, listOptions metav1.ListOptions) ([]*MCPBuilder, error) {
+	glog.V(100).Infof("Listing all MCP resources with the options %v\"", listOptions)
```

Comment:
Did you mean to leave this quote in?
```suggestion
	glog.V(100).Infof("Listing all MCP resources with the options %v", listOptions)
```